components/brave_ads/common/notification_ad_feature.h

```diff
@@ -13,9 +13,12 @@
 namespace brave_ads {
 
 BASE_DECLARE_FEATURE(kNotificationAdFeature);
+BASE_DECLARE_FEATURE(kAllowedToFallbackToCustomNotificationAdFeature);
```

Comment:
Rather than a feature, your thoughts on this being a param/value?

Reply:
Introducing `param/value` here requires additional patching of chromium's `about_flags.cc`. I don't think it's worth the effort here because our possible values are limited to `true/false` which is well covered by enabling/disabling a feature.

Reply:
Let me know when free to discuss more, I just need to jump on another call right now. Thanks